### ISSUE TITLE
Measuring Accuracy

### ISSUE THREAD

**Issue by sagedemage** [sagedemage/movie_recommendation_system#9]
Modify the training model program to measure the accuracy of the model during the evaluation of the model.

**Extra Tasks**
1. Fix the `train_one_epoch` function where it should be iterating to all of the batches of the training loader. When this function finishes, it should return the correct average loss. (bug)
2. Tweak SGD Optimization
    1. Tweak the epochs, and learning rate.
    2. Set the momentum of the optimizer to 0.9.
3. Modify the script to measure the accuracy of the movie recommendation program 1000 times.
4. Add links to resources that were useful for this project.

## Resource
- https://pytorch.org/tutorials/beginner/basics/optimization_tutorial.html#full-implementation

**Comment by sagedemage** [comment]
```
git fetch origin
git checkout 9-measuring-accuracy
```
